edreader/modulesInfo.go

```diff
@@ -0,0 +1,67 @@
+package edreader
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+)
+
+const FileModulesInfo = "ModulesInfo.json"
+
+// ModulesInfo struct to load the ModulesInfo file saved by ED
+type ModulesInfo struct {
+	Modules []ModulesLine
+}
+
+// ModulesLine struct to load individual module in the ModuleInfo
+type ModulesLine struct {
+	Slot        string
+	Item        string
+}
+
+var currentModules ModulesInfo
+
+func handleModulesInfoFile(file string) {
+	data, err := ioutil.ReadFile(file)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Please use
```
import (log "github.com/sirupsen/logrus")
```
and 
```
log.Errorln(err)
``` 
to match error logging in the rest of the project.

I see that I've used this same check in `cargo.go`, but that one should also be using the logging framework, I just missed it when I switched my error logging.

Reply:
Ok, I will update it